app/scripts/controllers/workflowdetails.js

```diff
@@ -24,6 +24,8 @@ angular.module('dockstore.ui')
       $scope.invalidClass = false;
       $scope.showEditWorkflowPath = true;
       $scope.showEditDescriptorType = true;
+      $scope.pathExtensions = ['cwl','wdl','yml','yaml'];
+
       if (!$scope.activeTabs) {
         $scope.activeTabs = [true];
         for (var i = 0; i < 3; i++) $scope.activeTabs.push(false);
```

Comment:
This '3' is a magic variable. ( https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad )
I know it wasn't part of this pull request, but could you break this out and give it an explanatory name?